build.gradle

```diff
@@ -862,6 +922,7 @@ task buildAll {
   dependsOn getTasksByName('product', true).collect { it.path }
   dependsOn getTasksByName('testClasses', true).collect { it.path }
   dependsOn distTar, distZip
+  dependsOn distInstallers
```

Comment:
Addendum: product dependency is not explicitly required since it is already done before "getTaskByName('product', true)..."